example.dae

```diff
@@ -5,6 +5,14 @@ global {
     # In normal case, you do not need to use it.
     tproxy_port: 12345
 
+    # Set it true to protect tproxy port from unsolicited traffic. Set it false to allow users to use self-managed
+    # iptables tproxy rules.
+    tproxy_port_protect: true
```

Comment:
If users don't specify this flag, what will happen?

Reply:
All value in example.dae is default value. That is to say, by default, tproxy_port_protect is true and reject traffic not from ebpf.